src/ngMock/angular-mocks.js

```diff
@@ -2249,6 +2250,47 @@ angular.mock.$ComponentControllerProvider = ['$compileProvider', function($compi
  *
  * <div doc-module-components="ngMock"></div>
  *
+ * @installation
+ *
+ *  <p>First, download the file:</p>
+ *  <ul>
+      <li>
+        <a href="https://developers.google.com/speed/libraries/devguide#angularjs">Google CDN</a> e.g.
+        {$ ("//ajax.googleapis.com/ajax/libs/angularjs/X.Y.Z/" + doc.packageFile) | code $}
+      </li>
+      <li>
+        <a href="https://www.npmjs.com/">NPM</a> e.g.
+        {% code %}npm install {$ doc.packageName $}@X.Y.Z{% endcode %}
+      </li>
+      <li>
+        <a href="http://bower.io">Bower</a><br> e.g.
+        {% code %}bower install {$ doc.packageName $}@X.Y.Z{% endcode %}
+      </li>
+      <li>
+        <a href="https://code.angularjs.org/">code.angularjs.org</a> (discouraged for
+        production use)  e.g.
+        {% code %}"//code.angularjs.org/X.Y.Z/{$ doc.packageFile $}"{% endcode %}
+      </li>
+    </ul>
+    <p>where X.Y.Z is the AngularJS version you are running.</p>
+
+    <p>Then, configure your test runner to load `angular-mocks.js` after `angular.js`.
+    This example uses <a href="http://karma-runner.github.io/">Karma</a>:</p>
+
+    {% code %}
+        config.set({
+          files: [
+            'build/angular.js', // and other module files you need
+            'build/angular-mocks.js',
+            '<path/to/application/files>',
+            '<path/to/spec/files>'
+          ]
+        });
+    {% endcode %}
+
+    <p>Including the `angular-mocks.js` file automatically adds the `ngMock` module, so your tests are ready to go!</p>
```

Comment:
It was just short enough that I didn't have to scroll, so I missed it :disappointed_relieved: 
Good thing you are watching :stuck_out_tongue: